lib/streamlit/server/Server.py

```diff
@@ -352,6 +352,10 @@ def is_ready_for_browser_connection(self):
     def browser_is_connected(self):
         return self._state == State.ONE_OR_MORE_BROWSERS_CONNECTED
 
+    @property
+    def is_running_hello(self):
+        return self._command_line.split(' ')[1].lower() == 'hello'
```

Comment:
Not sure what's better, checking command_line or script path but this looks nicer?

```
from streamlit.hello import hello
return self._script_path == hello.__file__
```

Reply:
done !, thanks karrie